client/state/plugins/wporg/test/actions.js

```diff
@@ -77,4 +77,17 @@ describe( 'WPorg Data Actions', function() {
 		WPorgActions.fetchPluginData( 'test' )( function() { } );
 		assert.equal( mockedWporg.getActivity().fetchPluginInformation, 1 );
 	} );
+
+	describe( 'Plugin lists', function() {
+		it( 'Actions should have method fetchPluginData', function() {
+			assert.isFunction( WPorgActions.fetchPluginsList );
+		} );
+
+		it( 'fetchPluginsList action should make a request', function( done ) {
+			WPorgActions.fetchPluginsList( 'test', 0 )( testDispatch( function() {
+				assert.equal( mockedWporg.getActivity().fetchPluginsList, 1 );
+				done();
+			}, 2 ) );
+		} );
+	} );
```

Comment:
Yeah, you're right! let's add them in another PR to not further complicate this one :+1: